ratis-assembly/src/main/assembly/examples-bin.xml

```diff
@@ -68,12 +68,12 @@
       <fileMode>755</fileMode>
     </fileSet>
     <!-- Include dev support tools -->
-    <fileSet>
-      <directory>${project.basedir}/../dev-support</directory>
-      <outputDirectory>dev-support</outputDirectory>
-      <fileMode>0644</fileMode>
-      <directoryMode>0755</directoryMode>
-    </fileSet>
+<!--    <fileSet>-->
+<!--      <directory>${project.basedir}/../dev-support</directory>-->
+<!--      <outputDirectory>dev-support</outputDirectory>-->
+<!--      <fileMode>0644</fileMode>-->
+<!--      <directoryMode>0755</directoryMode>-->
+<!--    </fileSet>-->
```

Comment:
Why commenting all these lines?  For debugging?

Reply:
hi @szetszwo, the `dev-support` should be put in source pkg. http://archive.apache.org/dist/ratis/2.3.0/apache-ratis-2.3.0-src.tar.gz

Reply:
Close the comment now.